runtime/src/swam/runtime/Engine.scala

```diff
@@ -163,6 +163,12 @@ object Engine {
       conf <- ConfigSource.default.at("swam.runtime").loadF[F, EngineConfiguration]
     } yield new Engine[F](conf, validator, None)
 
+  def apply[F[_]: Effect](tracer: Option[Tracer]): F[Engine[F]] =
```

Comment:
I would not add this method, but add the parameter to the one above, and make default value `None`.